ui/transformation.py

```diff
@@ -66,47 +55,61 @@ def setupUi(self, Transformation: "EditTransformation"):
             "Value to use in 3D view:", self.value_spinbox
         )
 
-        self.main_layout.addLayout(self.ui_placeholder_layout)
+        self.setup_ui(Transformation)
 
+    def setup_ui(self, transformation):
```

Comment:
Though this is the right thing to do, setupUI is used everywhere else in the code.
I suggest we leave it with the old name for now, and then do a PR where we change it everywhere at once.